### DISCUSSION TITLE
ScriptReference/RuntimeInitializeLoadType.SubsystemRegistration

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#538] [Notes]
# ScriptReference/RuntimeInitializeLoadType.SubsystemRegistration

https://docs.unity3d.com/ScriptReference/RuntimeInitializeLoadType.SubsystemRegistration.html

**Comment by joethephish** [comment]
Can be used to reset static fields and event handlers when you have Domain Reloading turned off, according to [this docs page on Domain Reloading](http://speace.chenjianqiu.ltd/unity2019_3/Manual/DomainReloading.html):

When I tested, it fired when entering play mode (but not on exit), and before my Awake call.
